File: testprojects/csharp/benchmark/domain/locationbased/latitude/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.socialmedia.locationbased.latitude;

namespace CSharpBenchmark.domain.locationbased.latitude
{

    //Functional requirement 3.2
    //Test case 115: Class domain.locationbased.latitude.Account is not allowed to use class infrastructure.socialmedia.locationbased.foursquare.AccountDAO 
    //Result: TRUE
    public class Account
    {
        public Account()
        {
            //FR5.1
            new AccountDAO();
        }
    }
}

File: testresources/java/benchmark_2012/domain/locationbased/foursquare/Account.java
package domain.locationbased.foursquare;

import infrastructure.socialmedia.locationbased.foursquare.AccountDAO;
//Functional requirement 3.2
//Test case 116: Class domain.locationbased.foursquare.Account is not allowed to use class infrastructure.socialmedia.locationbased.foursquare.AccountDAO 
//Result: FALSE
public class Account {
	public Account(){
		//FR5.1
		new AccountDAO();
	}
}